### DISCUSSION TITLE
Configuring Mimir Ruler Host

### DISCUSSION THREAD

**Discussion by chriskuchin** [grafana/grafana#55051] [Alerting]
So whenever I navigate to the alert rules page there is an error about the ruler api not being enabled. 

However when testing the datasource it successfully detects the ruler api. I am not clear on what it is basing the various detections on.

I have also attempted to upload a rule directly to the api to see if it's just getting an empty list 404 but that didn't clear it up either, and the alert did work so the ruler was active and I could list the alert via the api.

I have deployed mimir with some custom Terraform/ECS code so my theory is that whatever url that grafana expects the ruler to be at is different in my environment and I can't seem to find a config to update it

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
